### PR TITLE
Feature/lead onlines

### PR DESCRIPTION
**Pull request by hellangle**

### CODE REVIEW

File: src/Jarboo.Monera.Prototype.Websites/Services/LeadService.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+        public bool CanSee(string buyerId, string userId)
+        {
+            if (buyerId == userId) return true;
+            return _db.LeadBuyers.SingleOrDefault(u => u.BuyerId == buyerId && u.LeadId == userId) != null;
+        }
+
+        public bool CanBuy(string buyerId, string userId)
+        {
+            if (buyerId != userId) return true;
```
Comment (vgoloviznin) [inline src/Jarboo.Monera.Prototype.Websites/Services/LeadService.cs]: What does this `if` mean?

File: src/Jarboo.Monera.Prototype.Websites/Services/LeadService.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            return _db.LeadBuyers.SingleOrDefault(u => u.BuyerId == buyerId && u.LeadId == userId) == null;
```
Comment (vgoloviznin) [inline src/Jarboo.Monera.Prototype.Websites/Services/LeadService.cs]: You need also check if the number of buyers exceeds `ConfigurationManager.AppSettings["MaximumLeadBuyers"]`

**Comment by hellangle** [comment]
That is requirement:
"Add a new Console app, that upon running will get a list of property names and output it as a TXT file"

File: src/Jarboo.Monera.Prototype.Websites/Services/LeadService.cs (hunk 1 above)
Comment (hellangle) [inline src/Jarboo.Monera.Prototype.Websites/Services/LeadService.cs]: in CanSee method, it means the buyers can see theirself
In CanBuy method, it means the buyers can not buy theirself

**Pull request merged**